src/settings_internal.js

```diff
@@ -224,3 +224,6 @@ var LINK_AS_CXX = 0;
 // minimum set of ES6 featurs.  This triggers transpilation to ES5
 // using closure compiler.
 var TRANSPILE_TO_ES5 = 0;
+
+// Synthetic attribute which is union of the INCOMING_MODULE_JS_API and ADDITIONAL_MODULE_JS_API
```

Comment:
`ADDITIONAL_MODULE_JS_API` is just a plan at this stage? I don't see other references.

Reply:
Yeah.. coming in a followup PR.  Updated comment.